examples/validation_tests.ipynb

```diff
@@ -0,0 +1,558 @@
+    "cor_factor = clmm.utils._patch_rho_crit_to_cd2018(2.77533742639e+11)\n",
+    "omega_m = cosmo_clmm.get_E2Omega_m(z)*cor_factor\n",
+    "\n",
+    "rhocrit_mks = 3.*100.*100./(8.*np.pi*const.GNEWT.value)\n",
+    "rhocrit_cosmo = rhocrit_mks * 1000. * 1000. * const.PC_TO_METER.value * 1.e6 / const.SOLAR_MASS.value\n",
+    "\n",
+    "# Need to defined the 3d density and the tabulated sigma profile on a wide enough range\n",
+    "r_for_sigma = np.logspace(-6,2, len(r)*1000)\n",
+    "r3d = np.logspace(-7,3, len(r)*1000)\n",
+    "\n",
+    "# CT NFW through CT (numerical)\n",
+    "rho_nfw = ct.density.rho_nfw_at_r(r3d*cosmo_clmm['h'], Mvir*cosmo_clmm['h'], cvir, omega_m, delta=Delta)\n",
+    "integrand_nfw = rho_nfw/(omega_m*rhocrit_cosmo) # xi_nfw + 1 (see CT documentation)\n",
+    "\n",
+    "ct_Sigma_nfw_num = ct.deltasigma.Sigma_at_R(r*cosmo_clmm['h'], r3d*cosmo_clmm['h'], integrand_nfw, \n",
+    "                                        Mvir*cosmo_clmm['h'], cvir, omega_m)*cosmo_clmm['h']*1.0e12 # Msun/Mpc2\n",
+    "\n",
+    "# Redefine Sigma on finer grid for the numerical integration for DeltaSigma\n",
+    "tmp_nfw = ct.deltasigma.Sigma_at_R(r_for_sigma*cosmo_clmm['h'], r3d*cosmo_clmm['h'], integrand_nfw, \n",
+    "                                        Mvir*cosmo_clmm['h'], cvir, omega_m)*cosmo_clmm['h']*1.0e12 # Msun/Mpc2\n",
+    "ct_DeltaSigma_nfw_num = ct.deltasigma.DeltaSigma_at_R(r*cosmo_clmm['h'], r_for_sigma*cosmo_clmm['h'], tmp_nfw, \n",
+    "                                                  Mvir*cosmo_clmm['h'], cvir, omega_m)\n",
+    "\n",
+    "# CT Einasto through CT\n",
+    "rho_einasto = ct.density.rho_einasto_at_r(r3d*cosmo_clmm['h'], Mvir*cosmo_clmm['h'], cvir, alpha, omega_m, delta=Delta)\n",
+    "integrand = rho_einasto/(omega_m*rhocrit_cosmo) # xi_einasto + 1 (see CT documentation)\n",
+    "\n",
+    "ct_Sigma_ein = ct.deltasigma.Sigma_at_R(r*cosmo_clmm['h'], r3d*cosmo_clmm['h'], integrand, \n",
+    "                                        Mvir*cosmo_clmm['h'], cvir, omega_m)*cosmo_clmm['h']*1.0e12 # Msun/Mpc2\n",
+    "\n",
+    "\n",
+    "# Redefine Sigma on finer grid for the numerical integration for DeltaSigma\n",
+    "tmp = ct.deltasigma.Sigma_at_R(r_for_sigma*cosmo_clmm['h'], r3d*cosmo_clmm['h'], integrand, \n",
+    "                                        Mvir*cosmo_clmm['h'], cvir, omega_m)*cosmo_clmm['h']*1.0e12 # Msun/Mpc2\n",
+    "ct_DeltaSigma_ein = ct.deltasigma.DeltaSigma_at_R(r*cosmo_clmm['h'], r_for_sigma*cosmo_clmm['h'], tmp, \n",
+    "                                                  Mvir*cosmo_clmm['h'], cvir, omega_m)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## NFW: compare NC (analytical and numerical), CCL (numerical), CT (analytical and numerical) to analytical CCL"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "plt.figure()\n",
```

Comment:
can remove this line since the figure is declared in the next line with plt.subplots

Reply:
Done.